### ISSUE TITLE
Client will not reconnect to server

### ISSUE THREAD

**Issue by sfunk1x** [nextcloud/client_theming#38]
Thanks for reporting issues back to Nextcloud!

Please report all not theming related issues at the moment to https://github.com/owncloud/client

**FYI - I reported this at owncloud/client and the bug was immediately closed. Pretty sure because I'm using the nextcloud client, that this belongs with nextcloud and not owncloud...**
# 

Pardon me putting this bug here. The nextcloud client page suggested these bugs go here as opposed to under the nextcloud project (??). This appeared to break, for me, between Nextcloud 9 and 10.
Expected behaviour

When I closed and re-launch the Nextcloud client, I expect it to reconnect to the Nextcloud server the way my Android phone does.
Actual behaviour

The client can't reconnect to the server unless the account is removed from the client and then re-added. If that action is performed, then the account will sync. Closing the client and re-starting it (as in an update, or a reboot) will cause the client to fail to connect with a "credentials issue".
Steps to reproduce

```
Build the nextcloud client and launch.
Add account to nextcloud server.
Sync files (save a file to the sync folder, note that the icon changes and the log shows that files are synced)
Close client and relaunch.
```

Note that the client won't reconnect to the server
Server configuration

Operating system:

CentOS 7

Web server:

Nginx

Database:

PHP version:

7

ownCloud version:

10
Client configuration

Client version:

Current master

Operating system:

Ubuntu 16.04.01
Logs

10-21 19:49:56:701 1062 OCC::AbstractNetworkJob::start: !!! OCC::CheckServerJob created for "https://cloud.domain.com" + "status.php" "OCC::ConnectionValidator"
10-21 19:49:56:850 1062 OCC::CheckServerJob::finished: status.php returns: QMap(("edition", QVariant(QString, ""))("installed", QVariant(bool, true))("maintenance", QVariant(bool, false))("version", QVariant(QString, "9.1.0.16"))("versionstring", QVariant(QString, "10.0"))) QNetworkReply::NetworkError(NoError) Reply: QNetworkReplyHttpImpl(0x1e069c0)
10-21 19:49:56:851 1062 OCC::ConnectionValidator::slotStatusFound: *\* Application: ownCloud found: QUrl("https://cloud.domain.com/status.php") with version "10.0" ( "9.1.0.16" )
10-21 19:49:56:851 1062 OCC::AccountState::slotConnectionValidatorResult: AccountState connection status change: "Undefined" -> "Credentials Wrong"

```
Web server error log:
```

No errors recorded in nginx log.

```
Server logfile: ownCloud log (data/owncloud.log):
```

Exception: {"Message":"HTTP\/1.1 401 Username or password was incorrect, Username or password was incorrect","Exception":"Sabre\DAV\Exception\NotAuthenticated","Code":0,"Trace":"#0 [internal function]: Sabre\DAV\Auth\Plugin->beforeMethod(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))\n#1 \/usr\/share\/nginx\/html\/nextcloud\/3rdparty\/sabre\/event\/lib\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\n#2 \/usr\/share\/nginx\/html\/nextcloud\/3rdparty\/sabre\/dav\/lib\/DAV\/Server.php(446): Sabre\Event\EventEmitter->emit('beforeMethod', Array)\n#3 \/usr\/share\/nginx\/html\/nextcloud\/3rdparty\/sabre\/dav\/lib\/DAV\/Server.php(248): Sabre\DAV\Server->invokeMethod(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))\n#4 \/usr\/share\/nginx\/html\/nextcloud\/apps\/dav\/lib\/Server.php(184): Sabre\DAV\Server->exec()\n#5 \/usr\/share\/nginx\/html\/nextcloud\/apps\/dav\/appinfo\/v2\/remote.php(30): OCA\DAV\Server->exec()\n#6 \/usr\/share\/nginx\/html\/nextcloud\/remote.php(165): require_once('\/usr\/share\/ngin...')\n#7 {main}","File":"\/usr\/share\/nginx\/html\/nextcloud\/3rdparty\/sabre\/dav\/lib\/DAV\/Auth\/Plugin.php","Line":188,"User":false}

**Comment by JorgeGama** [comment]
I had a similar problem as well. I would recommend removing in files installed through "make install" and then follow this guide: https://github.com/nextcloud/client_theming/issues/25#issuecomment-254232998

It worked for me.

**Comment by sfunk1x** [comment]
That didn't really work for me. What the actual issue was that the Nextcloud server had been treating my login attempts as bruteforce an had recorded them in the db. Once I flushed the requisite IPs from that table, I was able to login. I'm guessing something happened in the switch from OwnCloud->NextCloud server AND switching from OwnCloud to Nextcloud client, on my end. I believe I whitelisted the ip range and configured a specific IP as my reverse proxy - now, I have no issues.
